installers/rpm-code/codes/flash.sh

```diff
@@ -1,33 +1,27 @@
 #!/bin/bash
 
 flash_main() {
+    local f="FLASH4.6.2"
     codes_dependencies hypre
-    codes_download_proprietary "flash/FLASH-4.6.2.tar.gz" "FLASH4.6.2"
+    codes_download_proprietary "flash/FLASH-4.6.2.tar.gz" "$f"
```

Comment:
reuse $f in both places.

Reply:
This required re-creating FLASH-4.6.2.tar.gz (dir contained within contained no `-` between FLASH and 4.6.2). You can find it in my homedir on you know what machine.

0fde278